File: JAVASCRIPT/EASY_JS_LEETCODE/PROB_1137_Nth_Tribonachi_Number.js
/**
 * @param {number} n
 * @return {number}
 */
var tribonacci = function(n) {
    if(n==0) return 0;
    let t = [0, 1 , 1];
    for(let i = 0 ; i < n-2 ; i++){
        t[i+3] = t[i] + t[i+1] + t[i+2];
    }
    return t[t.length-1]; 
};

File: C/C_DSA_QS_EASY/PROB_1137_Nth_Tribonachi_Number.c
#include <stdio.h>
int tribonacci(int n) {
    if(n==0){
        return 0;
    }
    int t[38] = {0 , 1 , 1};
    for(int i = 0 ; i < n - 2; i++){
        t[i+3] = t[i] + t[i+1] + t[i+2];
    }
    return t[n];
}